### ISSUE TITLE
Change "Next" to "Done" on Android's keyboard

### ISSUE THREAD

**Issue by mimf** [mpcjanssen/simpletask-android#649]
I recently came across this app and there's something that makes it borderline unsuable for me. 

When I've it in landscape mode, I can't see the "Save" button, so there's no easy way for me to save a task after I'm done typing. The keyboard has the "Next" key, but has no "done" key, which makes no sense to me. The "next" key gives a space, but there's already a space-bar on the keyboard, there's no need for that key, unless I'm missing something?

So I suggest changing the "Next" key to a "Done" key.

**Comment by smichel17** [comment]
I can verify this. For clarification: the enter key should still be next, but the large button on the right of the text area should close the keyboard.

**Comment by mimf** [comment]
Yes, I suppose that also works. Either one of them (or both really) should be a "Done/Save" key.

**Comment by smichel17** [comment]
The enter key is for adding multiple tasks at the same time! Which doesn't look quite right in landscape either, but is super useful.

**Comment by mimf** [comment]
Oh, I see. In landscape, all I was getting is a space, but after changing to portrait, there's a linebreak.

Well, then I guess button on the right side should be a done/save key.

**Comment by knitterb** [comment]
I definitely use the "next" to enter mutliple tasks and save considerable time...only wish when entering mutliple tasks that the list/context and project/tag were replicated all the way down. Please don't remove this, or make it a "multi-add" settings option. (Reference the Multi-Add functionality/setting in MyFitnessPal). Ping me on Gitter if you need more explanation of the Multi-Add functionality.

**Comment by mimf** [comment]
@knitterb, if you notice @smichel17's post, you'll see that the idea is to change one of the buttons only, as its behaviour in landscape is different than when in portrait, which should not happen. 

You can always use the "physical" button on the keyboard to chage line.

**Comment by knitterb** [comment]
@mimf You know what, you're right!  I didn't look at the UI when in landscape more until just now!  Sorry!  I've included a screenshot of the two duplicated buttons when in Landscape Mode so that we're clear of the duplication of actions.

![screenshot_20161025-111659](https://cloud.githubusercontent.com/assets/166302/19698635/f1633aee-9aa4-11e6-9412-ce1594426dd4.png)

**Comment by mpcjanssen** [comment]
I am not sure if this can actually be changed. I think it's the standard behaviour of the TextEdit in landscape mode.

**Comment by smichel17** [comment]
Updated, clarified thoughts:

![landscape-mode-clarified](https://cloud.githubusercontent.com/assets/6563664/19705104/ba4aff54-9ad8-11e6-9ff0-baf7d56fce37.png)

@mpcjanssen I was looking into this when it was originally posted. It is the standard behavior but it is customizable (with some difficulty).
